.github/workflows/test.yml

```diff
@@ -0,0 +1,62 @@
+name: Test
+on: [push]
+  # pull_request:
+  #   branches: [ master ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports: ["5432:5432"]
+        env:
+          POSTGRES_PASSWORD: postgres
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+    - run: ruby -v
+    
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.18.x'
+        registry-url: 'https://registry.npmjs.org'
+    
+    - name: Cache Gems
```

Comment:
[suggestion] The `Setup Ruby` step can do this for us. See https://github.com/OHS-Hosting-Infrastructure/complaint-tracker/blob/e9452643d5d672c47a94fe9e44b86bf09bcf33ef/.github/actions/setup-project/action.yml#L10 for an example.